### ISSUE TITLE
IDE: Error: ENOENT: no such file or directory (Windows path issue?)

### ISSUE THREAD

**Issue by Tobbe** [redwoodjs/redwood#1356]
When I opened VS Code today I was greeted by this
![image](https://user-images.githubusercontent.com/30793/96035500-64c33d00-0e63-11eb-8b91-3f63042d362b.png)

"Go to output" showed me this
```
Redwood.js Language Server onInitialize(), PID=8964
Redwood.js Language Server onInitialized()
(node:8964) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, scandir '\c%3A\Users\tobbe\dev\redwood\acm-store\web\src\pages\'
    at Object.readdirSync (fs.js:854:3)
    at Object.fs.readdirSync (electron/js2c/asar.js:621:39)
    at processPagesDir (c:\Users\tobbe\dev\redwood\acm-store\node_modules\@redwoodjs\internal\dist\paths.js:152:31)
    at RWProject.get processPagesDir (c:\Users\tobbe\dev\redwood\acm-store\node_modules\@redwoodjs\structure\dist\model\RWProject.js:125:39)
    at Object.getterCommon (c:\Users\tobbe\dev\redwood\acm-store\node_modules\lazy-get-decorator\common.js:23:32)
    at RWProject.get (c:\Users\tobbe\dev\redwood\acm-store\node_modules\lazy-get-decorator\legacy.js:19:29)
    at RWProject.get pages (c:\Users\tobbe\dev\redwood\acm-store\node_modules\@redwoodjs\structure\dist\model\RWProject.js:131:47)
    at Object.getterCommon (c:\Users\tobbe\dev\redwood\acm-store\node_modules\lazy-get-decorator\common.js:23:32)
    at RWProject.get (c:\Users\tobbe\dev\redwood\acm-store\node_modules\lazy-get-decorator\legacy.js:19:29)
    at RWProject.children (c:\Users\tobbe\dev\redwood\acm-store\node_modules\@redwoodjs\structure\dist\model\RWProject.js:87:39)
(node:8964) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 6)
```

That's just a snippet of the full output. Here's the whole thing: https://gist.github.com/Tobbe/0185f0942d8b405c7054303da964f43f

Looks like a Windows path issue
@aldonline

**Comment by aldonline** [comment]
Hi Tobbe,
This was fixed in the latest 0.19.x patch release.

**Comment by aldonline** [comment]
Closing for now. If you can reproduce with the latest 0.19.x, please reopen ;)
